cdci_data_analysis/analysis/parameters.py

```diff
@@ -806,3 +865,28 @@ def __init__(self, value=None, name_format='str', name=None):
     @staticmethod
     def check_name_value(value, units=None, name=None, par_format=None):
         pass
+
+class Boolean(Parameter):
+    owl_uris = ['http://www.w3.org/2001/XMLSchema#bool']
+    
+    def __init__(self, value=None, name=None):
+
+        self._true_rep = ['True', 'true', 'yes', '1', True]
+        self._false_rep = ['False', 'false', 'no', '0', False]
+        super().__init__(value=value,
+                         name=name,
+                         allowed_values=self._true_rep+self._false_rep
+                         )
+
+    @property
+    def value(self):
+        return str(self._value).lower() #because passed in json
```

Comment:
always in json?

EDIT: you mean that it always will be used when building a json object?

Reply:
Good question. This came from gw plugin (I added boolean parameter locally there).
There, it's used this way. And it conforms to the way other parameters are passed.

On the other hand, the general logic is to return real boolean value. So I'm not sure on this.

Probably, it's better be postprocessed in custom encoder. I'll try